release

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
There's no bash-specific code here, so we can just use `#!/bin/sh`.

Reply:
Good idea! Shell scripting is not my strong suite for sure 😬